test/presentation/agenda/agenda_view_model_test.dart

```diff
@@ -190,6 +190,19 @@ void main() {
     });
   });
 
+  group('isReloading', () {
```

Comment:
A voir, côté web c'est pas mal fait, mais je crois que sur notre projet app on met rarement de group pour un seul test. Ou alors tu mets un autre test dans le cas false.

Reply:
Il y a une pratique qu'on fait à plusieurs endroits (consciemment ou inconsciemment je sais pas), c'est d'avoir des groupes quand on teste des choses spécifiques / une property précise de la classe. J'aime bien l'idée. 

Là effectivement il n'y a qu'un test, et c'était peut-être pas fait ailleurs, mais au moins on structure un peu, ça se lit bien : "dans le contexte de la property Y, on teste que when blabla should blabla".  Et les "tests à false" existent déjà idnirectement parce que ce sont les cas nomianux, testés dans les plus grands groupes. Ca ferait doublon de refaire un false je pense.